graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/processor/InsertModelBuilder.java

```diff
@@ -77,11 +82,15 @@ InsertModel build() throws SkipException {
           operationName, entity.getGraphqlName());
     }
 
+    // we are using GraphQLString because it will be coerced to BigInteger (Long)
+    Optional<String> cqlTimestampArgumentName =
+        findFieldNameWithDirective(
+            CqlDirectives.TIMESTAMP, Arrays.asList(Scalars.GraphQLString, Scalars.GraphQLInt));
```

Comment:
What if there is a second field, but it's not annotated with `@cql_timestamp`?

Reply:
Currently, it will be ignored (I added a test case for it). I am not sure if that's ideal. We can do 2 things:
1. Take and use the second parameter as the `writeTimestamp` even if it is not annotated with a `@cql_timestamp`. I think it is a bit risky - users may have accidentally updated write. Also, If we need to add another insert directive support, the ordering would be critical, and we don't want to do it.
2. If there is a second argument, but without any directive, we may throw an exception and report all possible directives supported on the insert. This option seems better to me.
wdyt?

Reply:
I agree 2 is better. Since there are only two possible arguments we can use a simple check like `inputs.size()==2 && ˜cqlTimestampArgumentName.isPresent()`.